src/shell/shell.cpp

```diff
@@ -880,7 +891,30 @@ void SHELL_Init() {
 	        "Examples:\n"
 	        "  \033[32;1mset\033[0m\n"
 	        "  \033[32;1mset\033[0m \033[37;1mname\033[0m=\033[36;1mvalue\033[0m\n");
-	MSG_Add("SHELL_CMD_IF_HELP","Performs conditional processing in batch programs.\n");
+	MSG_Add("SHELL_CMD_IF_HELP",
+	        "Performs conditional processing in batch programs.\n");
+	MSG_Add("SHELL_CMD_IF_HELP_LONG",
+	        "Usage:\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[36;1merrorlevel\033[0m \033[37;1mNUMBER\033[0m COMMAND\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[37;1mSTR1==STR2\033[0m COMMAND\n"
+	        "  \033[32;1mif\033[0m \033[35;1m[not]\033[0m \033[36;1mexist\033[0m \033[37;1mFILE\033[0m COMMAND\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[37;1mNUMBER\033[0m     is a positive integer less or equal to the desired value.\n"
+	        "  \033[37;1mSTR1==STR2\033[0m compares two text strings (case sensivitive).\n"
+	        "  \033[37;1mFILE\033[0m       is an exact file name to check for existence.\n"
+	        "  COMMAND    is a DOS command/program to run, optionally with parameters.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  The COMMAND will only run if any of the three conditions is true.\n"
+	        "  If \033[38;1mnot\033[0m is specified, then the command runs only with the false condition.\n"
```

Comment:
The first line mentions the command is only run on "true" conditions, but then this line says says the command is run "only with the false condition" - which is confusing.  Can it be reworded?

Reply:
I agree that it may be better to rephrase the statements so that it is even easier to read for beginners, but I am not exactly sure about the best way to reword this one. Indeed, it appears to be somewhat tricky to add help messages for this command.

Reply:
Thanks for getting me up to speed @Wengier.  Indeed, the wording is tricky - and your suggestions are helpful, so we should try to keep it all.  
Will take another pass through tomorrow! Regards.